nmt_chainer/evaluation.py

```diff
@@ -185,8 +185,10 @@ def beam_search_translate(encdec, eos_idx, src_data, beam_width = 20, beam_pruni
                 rescored_translations.append((tr, sc + reverse_scores[num_t], attn))
             translations = rescored_translations
                     
-        if score_is_divided_by_length:
+        if post_score_length_normalization == 'simple':
             translations.sort(key = lambda x:x[1]/(len(x[0])+1), reverse = True)
+        elif post_score_length_normalization == 'google':
+            translations.sort(key = lambda x:x[1]/( pow((len(x[0])+6), length_normalization_strength) / pow(6, length_normalization_strength) ), reverse = True)
```

Comment:
in the simple case, it was mostly to avoid division by zero if x[0] is an empty string (even if it is unlikely to happen in practice).